doc/api/stream.md

```diff
@@ -2162,8 +2162,19 @@ user programs.
 
 #### `writable._writev(chunks, callback)`
 
-* `chunks` {Object[]} The chunks to be written. Each chunk has following
-  format: `{ chunk: ..., encoding: ... }`.
+* `chunks` {Object[]} The data to be written. The value is an array of {Object}
+  that each represent a discreet chunk of data to write. The properties of
+  these objects are:
+  * `chunk` {Buffer|string} A buffer instance or string containing the data to
+    be written. The `chunk` will be a string if the `Writable` was created with
+    the `decodeStrings` option set to `false`.
+  * `encoding` {string} The character encoding of the `chunk`. If `chunk` is
+    a `Buffer`, the `encoding` will be `'buffer`.
+  * `callback` {Function} A reference to the write callback associated with this
+    chunk (passed in when the `write()` or `end()` function is called). It is
+    not intended for user code to access the callback directly.
+  * `next` {Object} A reference to the next chunk in the list. It is not intended
+    for user code to access the `next` object directly.
```

Comment:
Something not being intended for access is a really good reason not to document it.

Reply:
It's also a really good reason to deprecate the public accessible property and hide it behind an internal symbol ;) 

I've no problem not documenting these also.